conan/api/subapi/config.py

```diff
@@ -68,3 +71,47 @@ def global_conf(self):
                     """)
                 save(global_conf_path, default_global_conf)
         return self._new_config
+
+    @staticmethod
```

Comment:
Maybe a property0 Why settings_yaml is property and this is method? would be a bit non homogeneous UI.